package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "tdd-strict-equal",
+  "version": "1.0.0",
+  "description": "homework, tdd-strict-equal",
+  "main": ".eslintrc.js",
+  "scripts": {
+    "lint": "/node_modules/.bin/eslint --format codeframe .",
+      "test": "./node_modules/.bin/mocha ./tests/*.test.js"
```

Comment:
you created a test folder not a tests folder so you just want to be consistent. 
this should be `"test": "./node_modules/.bin/mocha ./test/*.test.js"`